app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
-  self.abstract_class = true
+  primary_abstract_class
+
+  def current_user
+    Users.first
+  end
```

Comment:
- [ ] This should be in the `applicationController` please change it and it should be 
`User.first`